dev-haskell/servant-client-core/servant-client-core-0.18.2.ebuild

```diff
@@ -35,5 +35,11 @@ RDEPEND=">=dev-haskell/aeson-1.4.1.0:=[profile?] <dev-haskell/aeson-1.6:=[profil
 DEPEND="${RDEPEND}
 	>=dev-haskell/cabal-1.24.2.0
 	test? ( >=dev-haskell/hspec-2.6.0 <dev-haskell/hspec-2.8
-		>=dev-haskell/quickcheck-2.12.6.1 <dev-haskell/quickcheck-2.14 )
+		>=dev-haskell/quickcheck-2.12.6.1 <dev-haskell/quickcheck-2.15 )
```

Comment:
Upper bound should be removed completely.

Reply:
My strategy for bumping dependencies has turned into:

1. check for hackage revisions
2. if present, re-run hackport to capture the new revisions
3. pull the updated cabal file from hackage
4. anything that still needs to be bumped, I will use `cabal_chdeps` and remove the upperbound

Notice that using this method, anything changed by upstream in the revision *will still contain an upperbound*. I think this is ok, and draws a clear distinction between what we have to "hack" to make things work, versus what upstream has already incorporated into their package.